yt/frontends/artio/data_structures.py

```diff
@@ -333,6 +333,25 @@ def _read_fluid_fields(self, fields, dobj, chunk=None):
         )
         return fields_to_return, fields_to_generate
 
+    def _icoords_to_fcoords(self, icoords, ires, axes=None):
```

Comment:
Base class *doesn't* implement the method.

Reply:
Oh that's right. Looks like it should be declared as an abstract method in the base class then, since it's not optional (unless I'm missing something again ?)

Reply:
done